provisioning/tasks/drupal.yml

```diff
@@ -13,14 +13,14 @@
 
 - name: Generate Drupal site with drush makefile.
   command: >
-    drush make -y {{ drush_makefile_path }} --no-gitinfofile
+    {{ drush_path }} make -y {{ drush_makefile_path }} --no-gitinfofile
```

Comment:
drush_path comes from the drush role. I had to use it on CentOS 6 because drush was not in the path.